app/metrics.yaml

```diff
@@ -372,6 +372,195 @@ metrics:
     - fenix-core@mozilla.com
     expires: "2019-09-01"
 
+# The `history_sync` and `bookmarks_sync` metrics use the same structure, but are
+# sent in separate pings, after each sync. This means we submit one ping per
+# store per sync.
+history_sync:
```

Comment:
In mozilla-mobile/android-components#3092 you mention budling `metrics.yaml` and `pings.yaml` in the sync component in a-c. I think that's the best approach, if these metrics and pings are always going to be the same for all products using sync. This avoids the burden of changing app code whenever something needs to be tweaked.

In addition to that, if you follow that path, to make data reviewers happy about your PR you should probably also have docs describing what telemetry your component is collecting. Maybe also mention that data-review will still be required for products using sync prior to start sending telemetry. @liuche definitely can tell better.

Reply:
Yes! The idea here for documentation is that consumers of your library would know what kinds of data collection they're sending by default (in using your library). This is also something that consumers of the library can link to so users have a clear chain of data collection.